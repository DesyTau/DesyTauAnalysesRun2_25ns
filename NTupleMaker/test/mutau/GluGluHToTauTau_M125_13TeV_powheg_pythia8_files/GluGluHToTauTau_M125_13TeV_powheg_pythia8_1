/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_34.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_35.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_36.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_37.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_38.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_39.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_40.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_41.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_42.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_43.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_0.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_1.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_100.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_101.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_102.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_103.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_104.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_105.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_106.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_107.root
