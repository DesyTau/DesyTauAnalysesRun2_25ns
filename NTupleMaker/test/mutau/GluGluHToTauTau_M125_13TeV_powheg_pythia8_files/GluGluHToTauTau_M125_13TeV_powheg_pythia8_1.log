000 (2509986.000.000) 01/17 11:47:04 Job submitted from host: <131.169.223.40:9618?addrs=131.169.223.40-9618+[2001-638-700-10df--1-28]-9618&noUDP&sock=7812_f5f0_32>
...
022 (2509986.000.000) 01/17 11:48:54 Job disconnected, attempting to reconnect
    Socket between submit and execute hosts closed unexpectedly
    Trying to reconnect to slot1@bird300.desy.de <131.169.76.38:35524?addrs=131.169.76.38-35524>
...
024 (2509986.000.000) 01/17 11:49:03 Job reconnection failed
    Job not found at execution machine
    Can not reconnect to slot1@bird300.desy.de, rescheduling job
...
001 (2509986.000.000) 01/17 11:49:38 Job executing on host: <131.169.164.38:38551?addrs=131.169.164.38-38551>
...
006 (2509986.000.000) 01/17 11:49:47 Image size of job updated: 157236
	100  -  MemoryUsage of job (MB)
	101772  -  ResidentSetSize of job (KB)
...
006 (2509986.000.000) 01/17 11:49:58 Image size of job updated: 157236
	100  -  MemoryUsage of job (MB)
	101892  -  ResidentSetSize of job (KB)
...
005 (2509986.000.000) 01/17 11:49:58 Job terminated.
	(1) Normal termination (return value 255)
		Usr 0 00:00:15, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:15, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	0  -  Run Bytes Sent By Job
	0  -  Run Bytes Received By Job
	0  -  Total Bytes Sent By Job
	0  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.00        1         1 
	   Disk (KB)            :     1     3072000   3856750 
	   Memory (MB)          :   100        2048      2048 
...
