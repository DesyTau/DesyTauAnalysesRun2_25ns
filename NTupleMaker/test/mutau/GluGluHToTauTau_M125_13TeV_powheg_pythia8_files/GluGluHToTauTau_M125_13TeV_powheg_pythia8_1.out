/nfs/dust/cms/user/cardinia/HtoTauTau/HiggsCP/CMSSW_9_4_9/src/DesyTauAnalyses/NTupleMaker/test/mutau/GluGluHToTauTau_M125_13TeV_powheg_pythia8_files
Number of single lepton trigger legs = 2
Number of X trigger legs (lep leg)   = 2
Number of X trigger legs (tau leg)   = 2
Number of ditau trigger legs         = 0
dxyLeptonCut 0.045
dzLeptonCut 0.2
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_34.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_35.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_36.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_37.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_38.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_39.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_40.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_41.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_42.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8/GluGluHToTauTau_M125_13TeV_powheg_pythia8_43.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_0.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_1.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_100.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_101.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_102.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_103.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_104.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_105.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_106.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_107.root
Histogram GluGluHToTauTau_M125_13TeV_powheg_pythia8_pileup is not present in pileup file
